src/main/java/org/folio/circulation/domain/OpeningPeriod.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.circulation.domain;
+
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.joda.time.LocalDate;
+import org.joda.time.LocalTime;
+
+public class OpeningPeriod {
+  private static final String OPENING_DAY_KEY = "openingDay";
+  private static final String DATE_KEY = "date";
+
+  private final OpeningDay openingDay;
+  private final LocalDate date;
+
+  public OpeningDay getOpeningDay() {
+    return openingDay;
+  }
+
+  public LocalDate getDate() {
+    return date;
+  }
+
```

Comment:
Why are these property methods above the constructor definition?

Reply:
Fixed.